include/SFML/Window/Event.hpp

```diff
@@ -276,6 +276,17 @@ class SFML_WINDOW_API Event
     template <typename T>
     [[nodiscard]] const T* getIf() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Check if current event type is not `Empty`
+    ///
+    /// \return True if current event type is not `Empty`
```

Comment:
I wonder if doxygen makes a link to the Empty struct when using backticks.

Reply:
I am not sure it is worth adding backticks actually. We do not usually do it so it would look inconsistent.

Reply:
I can understand; I usually use backticks in my personal code because most renderers will treat it as a line of code (aka a <code> element in HTML). It ends up looking nicer in my opinion. If for consistency you revert it, that's fine.

Reply:
> I am not sure it is worth adding backticks actually. We do not usually do it so it would look inconsistent.

It makes the documentation easier to browse, I think it's worth it assuming that there are going to be subsequent PRs to improve the existing docs.

Reply:
Leaving it as is sounds good to me.

Reply:
> > I am not sure it is worth adding backticks actually. We do not usually do it so it would look inconsistent.
> 
> It makes the documentation easier to browse, I think it's worth it assuming that there are going to be subsequent PRs to improve the existing docs.

I'd be down to make those PRs.